src/app/interactions/overlay/overlay-styling/overlay-styling.component.html

```diff
@@ -1,4 +1,4 @@
 <div class="combo-wrapper">
-  <igx-combo [data]="items" valueKey="name" displayKey="name" [ngModel]="[items[2]]" width="410px" [overlaySettings]="{ outlet: element, modal: true }">
+  <igx-combo [data]="items" valueKey="name" displayKey="name" [(ngModel)]="selectedItems" width="410px" [overlaySettings]="{ outlet: element, modal: true }">
```

Comment:
Instead of adding a new prop, isn't it better to simply add the path as follows:
```suggestion
  <igx-combo [data]="items" valueKey="name" displayKey="name" [(ngModel)]="[items[2].name]" width="410px" [overlaySettings]="{ outlet: element, modal: true }">
```